src/components/block-button.js

```diff
@@ -0,0 +1,31 @@
+window.blockUser = function(id) {
```

Comment:
I assume these were just global for debugging? Should make them local.

Reply:
👍